src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/ReleasePaymentModal/consts.ts

```diff
@@ -0,0 +1,42 @@
+import { ColonyRole } from '@colony/colony-js';
+
+import { UserRole, getRole } from '~constants/permissions.ts';
+import { useAppContext } from '~context/AppContext/AppContext.ts';
+import { useColonyContext } from '~context/ColonyContext/ColonyContext.ts';
+import { getAllUserRoles } from '~transformers';
+import { DecisionMethod } from '~types/actions.ts';
+import { formatText } from '~utils/intl.ts';
+
+import { type DecisionMethodOption } from '../DecisionMethodSelect/types.ts';
+
+export const useGetReleaseDecisionMethodItems = (): DecisionMethodOption[] => {
+  const { colony } = useColonyContext();
+  const { user } = useAppContext();
+
+  if (!user) {
+    return [];
+  }
+
+  const userPermissions = getAllUserRoles(colony, user.walletAddress);
+  const userRole = getRole(userPermissions);
+
+  const isPermissionsEnabled =
+    userRole.role === UserRole.Owner ||
+    userRole.role === UserRole.Admin ||
+    userRole.role === UserRole.Payer ||
+    userPermissions.includes(ColonyRole.Arbitration);
```

Comment:
It's sufficient just to check if the user has `Arbitration` permission but it must take into account the domain of the expenditure. You can replace it with a single check using `addressHasRoles`:
```ts
addressHasRoles({
  address: user.walletAddress,
  colony,
  requiredRoles: [ColonyRole.Arbitration],
  requiredRolesDomains: [expenditure.nativeDomainId] // will need to pass expenditure as prop
})
```